platform/windows/resource.hpp.in

```diff
@@ -0,0 +1,11 @@
+// Specify icon from resource file as icon of executable
```

Comment:
This file can be in directly in our include folder but maybe `include/inexor/vulkan-renderer/plaftorm/windows`

Reply:
Done.